NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty
# Post Create, Edit, Delete

**Description**
When you click on MyPosts, you can click the button on the top right to "Create New Post". You can fill out the title, subject, category, and hit "Save" and the post should update to your list of my posts. Similarly, a delete and edit button should be next to your posts. When you hit delete it will prompt you to delete the post and if you hit "Yes" it will be removed from your post list. You can hit the edit button and change one of the input fields, and hit save, it will change in your list of posts too.
[Trello Ticket #6](https://trello.com/c/V9SZgJhj)
[Trello Ticket #7](https://trello.com/c/d1C2B1hT)
[Trello Ticket #8](https://trello.com/c/aWov9G09)

**Testing Instructions**
- git add and git commit whatever you're working on on your branch
- git checkout main
- git fetch --a git checkout: at-editdeletepost
- If you don't see the most recent version of my files, git pull origin: at-editdeletepost
- If you haven't created the Tabloid MVC database, please do so.
- Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty/blob/main/SQL/01_Tabloid_Create_DB.sql)
- Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-9/tabloid-full-stack-debugging-dynasty/blob/main/SQL/02_Tabloid_Seed_Data.sql)
- Click the execute with debugger solid green button to launch the app
- In browser, please go to Login on the nav bar and log in with this email: [foo@bar.com](mailto:foo@bar.com)
- This should redirect you back to the home page.'You should see a "MyPosts" option on the navbar
- When you click on the "MyPosts" button it should redirect you to another page that displays the list of Posts that the current
user has made
- You can click the "Create New Post" button on the top right, fill out the fields (title, subject, category) and hit save, it will appear in your list of posts
- You can click the "Edit" button next to the post and change any of the input fields, hit save, and it will save to your list of posts
- You can hit the "Delete" button next to the post, it will prompt you if you are sure, you hit "Yes" and it will disappear from your list of posts

**Checklist:**
